ci/cloudbuild/dockerfiles/fedora-latest-publish-docs.Dockerfile

```diff
@@ -83,3 +71,14 @@ RUN curl -fsSL https://github.com/mozilla/sccache/releases/download/v0.7.7/sccac
 
 # Update the ld.conf cache in case any libraries were installed in /usr/local/lib*
 RUN ldconfig /usr/local/lib*
+
+# Install the Cloud SDK and some of the emulators. We use the emulators to run
+# integration tests for the client libraries.
```

Comment:
nit: this comment isn't relevant to the publish-docs build. Maybe just "Install the Cloud SDK" or "Install `gcloud`"

Reply:
Done and done.